src/appengine/handlers/reproduce_tool/get_config.py

```diff
@@ -16,7 +16,7 @@
 from future import standard_library
 standard_library.install_aliases()
 
-import urllib
+from urllib import parse
```

Comment:
we usually do "import urllib.parse" in other places so that a generic parse function can't conflict.